components/hab/src/cli/hab/origin.rs

```diff
@@ -235,6 +238,60 @@ pub struct UploadGroup {
     public_file: Option<PathBuf>,
 }
 
+#[derive(ConfigOpt, StructOpt)]
+#[structopt(no_version)]
+/// Commands related to origin rbac
+pub enum Rbac {
+    /// Display an origin member's current role
+    Show {
+        /// The account name of the role to display
+        #[structopt(name = "MEMBER_ACCOUNT")]
+        member_account: String,
+        /// The name of the origin for the origin member association
+        #[structopt(name = "ORIGIN",
+                short = "o",
+                long = "origin",
+                required = true,
+                validator = valid_origin)]
```

Comment:
:thinking: should we have an `Origin` type. It could help make the code more self document and we would not need to explicitly state this validator. We could just have `origin: Origin`.

Reply:
I have done so.